editor/i18n/en/localization.js

```diff
@@ -431,6 +538,40 @@ module.exports = {
         startRadius: 'Starting radius and variation of the particles. Only available in Radius mode',
         endRadius: 'Ending radius and variation of the particles. Only available in Radius mode',
         rotatePerS: 'Number of degress to rotate a particle around the source pos per second and variation. Only available in Radius mode',
+        capacity: 'Maximum particle amount that the system can generate',
+        scaleSpace: 'Scale space',
+        startSize3D: 'Set start size of X, Y and Z respectively if true',
+        startSizeX: 'Start size in X direction',
+        startSizeY: 'Start size in Y direction',
+        startSizeZ: 'Start size in Z direction',
+        startSpeed: 'Start speed',
+        startRotation3D: 'Set rotation of X, Y and Z respectively if true',
+        startRotationX: 'Angle of rotation around X-axis at beginning',
+        startRotationY: 'Angle of rotation around Y-axis at beginning',
+        startRotationZ: 'Angle of rotation around Z-axis at beginning',
+        startDelay: 'Time delayed of particle emit after the system start running',
+        startLifetime: 'Life time of the particle',
+        duration: 'Particle duration',
+        loop: 'Loop animation',
+        prewarm: 'Animation plays from the end state of last round(only valid when loop activated)',
+        simulationSpace: 'The particle space do your calculation in',
+        simulationSpeed: 'Update ratio of particles',
+        playOnAwake: 'Play on awake',
+        gravityModifier: 'How much particles get affected by the gravity',
```

Comment:
Only available in CPU particle system.

heck other property only available in CPU with @caryliu1999